README.md

```diff
@@ -36,6 +162,10 @@ npm run generate
 npm run deploy
 ```
 
+## Testing
+
+The Annotation tool uses [Cypress](https://www.cypress.io/) for integration, component, and unit testing. See the [cypress folder](https://github.com/neurobagel/annotation_tool/tree/master/cypress) for tests, custom commands, and test data.
+
 See the [Nuxt documentation](https://nuxtjs.org/deployments/github-pages/) for more details.
```

Comment:
Is this left over from the old README.md?

Reply:
Yes, and this should be under the deployment section. Moving it.